src/combobox/ComboBox.ts

```diff
@@ -169,19 +162,19 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 	private _onInputKeyDown(event: KeyboardEvent) {
 		const { results = [] } = this.properties;
 
-		switch (event.keyCode) {
-			case keys.up:
+		switch (event.which) {
```

Comment:
If we could use `event.key` I think it would make sense to just compare against `'ArrowDown'` vs. `keys.ArrowDown`. Might be moot though, unless we can ignore the Android browser and Safari 10.0/iOS Safari <10.3.